sentry-ruby/lib/sentry/scope.rb

```diff
@@ -7,7 +7,7 @@ module Sentry
   class Scope
     include ArgumentCheckingHelper
 
-    ATTRIBUTES = [:transaction_names, :contexts, :extra, :tags, :user, :level, :breadcrumbs, :fingerprint, :event_processors, :rack_env, :span, :session]
+    ATTRIBUTES = [:transaction_names, :contexts, :extra, :tags, :user, :level, :breadcrumbs, :fingerprint, :event_processors, :rack_env, :span, :session, :trace]
```

Comment:
Needed some way to get the Sentry Trace (the trace ID/span ID/sample flag we store in the `sentry-trace` header) from the Rack middleware that reads it to the SpanProcessor that actually uses it (to create the linked transaction). Alternatives very welcome.

Reply:
We store request env in `rack_env`, so perhaps reading the header from it could also work?

Reply:
done